packages/mermaid/src/diagrams/sequence/parser/sequenceDiagram.jison

```diff
@@ -84,7 +85,7 @@ accDescr\s*"{"\s*                                { this.begin("acc_descr_multili
 \-\-[x]                                                         return 'DOTTED_CROSS';
 \-[\)]                                                          return 'SOLID_POINT';
 \-\-[\)]                                                        return 'DOTTED_POINT';
-":"(?:(?:no)?wrap:)?[^#\n;]+                                    return 'TXT';
```

Comment:
Removing `#` here so that CSS colors (e.g. #FFC0CB) can be passed. 
Unsure why the `#` was here in the first place. Please double check if it's safe to remove it.